contracts/handlers/ERC20Handler.sol

```diff
@@ -33,14 +33,33 @@ contract ERC20Handler is IDepositHandler, ERC20Safe {
         _;
     }
 
-    constructor(address bridgeAddress) public {
+    constructor(address bridgeAddress, bytes[] memory initialTokenIDs, address[] memory initialContractAddresses) public {
+        require(initialTokenIDs.length == initialContractAddresses.length,
+            "mismatch length between initialTokenIDs and initialContractAddresses");
+
         _bridgeAddress = bridgeAddress;
+
+        for (uint256 i = 0; i < initialTokenIDs.length; i++) {
+            _tokenIDToTokenContractAddress[initialTokenIDs[i]] = initialContractAddresses[i];
+            _tokenContractAddressToTokenID[initialContractAddresses[i]] = initialTokenIDs[i];
+        }
     }
 
     function getDepositRecord(uint256 depositID) public view returns (DepositRecord memory) {
         return _depositRecords[depositID];
     }
 
+    function setTokenIDAndContractAddress(bytes memory tokenID, address contractAddress) public {
```

Comment:
This method assumes:

1. You shouldn't be able to set either the `tokenID` or contract address without setting the other
2. You shouldn't be allowed to overwrite existing `tokenID`s or contract addresses
3. Anyone can set a `tokenID` and contract address

I have reservations about 3, but don't know if we want to provide a full fledged fix for it now?

Reply:
Ya #3 is a bit scary... Ill need to think about it

Reply:
In the same way we have threshold proposals we could have tokenId proposals. 

This might be a good reason for trying to generalize our notion of proposal so that threshold proposals == deposit proposals == tokenId proposals, instead of them being separate. Maybe this is as simple as supporting the relayer contract as a new type of handler (for the case of relayer proposals). :thinking: Separating the registry from the handler (separate contract) might also open up possibilities.